charts/vela-core/values.yaml

```diff
@@ -88,19 +88,23 @@ healthCheck:
 ##@param optimize.markWithProb Optimize ResourceTracker GC by only run mark with probability. Side effect: outdated ResourceTracker might not be able to be removed immediately.
 ##@param optimize.disableComponentRevision Optimize componentRevision by disabling the creation and gc
 ##@param optimize.disableApplicationRevision Optimize ApplicationRevision by disabling the creation and gc.
-##@param optimize.disableWorkflowRecorder Optimize workflow recorder by disabling the creation and gc.
 ##@param optimize.enableInMemoryWorkflowContext Optimize workflow by use in-memory context.
 ##@param optimize.disableResourceApplyDoubleCheck Optimize workflow by ignoring resource double check after apply.
 ##@param optimize.enableResourceTrackerDeleteOnlyTrigger Optimize resourcetracker by only trigger reconcile when resourcetracker is deleted.
+##@param optimize.informerCache if enabled, informer cache will be optimized by removing unnecessary fields will be filtered
```

Comment:
can you explain what's the unnecessary fields? we need to answer if these fields are unnecessary, why don't make it default to true?

Reply:
We can make it default to be true. The unnecessary fields are the fields that are recorded in etcd storage but vela-core does not need to use them for logic computation, like `managedFields` or `last-applied-configuration`.

Reply:
if so, please don't expose a chart parameter to add complexity